docs/cpu_templates/schema.json

```diff
@@ -5,6 +5,32 @@
     "description": "Custom CPU template language description",
     "type": "object",
     "properties": {
+        "kvm_capabilities": {
+            "type": "array",
+            "items": {
+                "description": "Additional KVM capabilities can be added or existing (build in) capabilities can be removed from the firecracker checks. To add KVM capability to the check list specify decimal number of corresponding KVM capability. To remove KVM capability from the check list specify decimal number of corresponding KVM capability with '!' mark in the front. Works on both x86_64 and aarch64",
```

Comment:
s/check list/checklist/
s/build in/built in/
s/of corresponding/of the corresponding/
s/remove KVM capability/remove a KVM capability/

It would be nice to provide a link to the kernel doc/source where user can find the values.

Reply:
Don't think links in the `shema.json` make sense. Will add them in the `cpu-templates.md`